charts/karpenter-crd/templates/karpenter.azure.com_aksnodeclasses.yaml

```
@@ -484,6 +492,10 @@ spec:
                 pattern: (?i)^\/subscriptions\/[^\/]+\/resourceGroups\/[a-zA-Z0-9_\-().]{0,89}[a-zA-Z0-9_\-()]\/providers\/Microsoft\.Network\/virtualNetworks\/[^\/]+\/subnets\/[^\/]+$
                 type: string
             type: object
+            x-kubernetes-validations:
+            - message: FIPS is not yet supported for Ubuntu2204
+              rule: 'has(self.imageFamily) && has(self.fipsMode) ?  !(self.imageFamily
```

extremely minor, nit: I think you have two spaces here, which I don't think you need:
```suggestion
              rule: 'has(self.imageFamily) && has(self.fipsMode) ? !(self.imageFamily
```